content/docs/csidriver/installation/helm/powerstore.md

```diff
@@ -117,7 +117,22 @@ Use [v5.0.x](https://github.com/kubernetes-csi/external-snapshotter/tree/v5.0.1/
    - [quay.io/k8scsi/csi-snapshotter:v4.0.x](https://quay.io/repository/k8scsi/csi-snapshotter?tag=v4.0.0&tab=tags)
 - The CSI external-snapshotter sidecar is still installed along with the driver and does not involve any extra configuration.
 
-## Volume Health Monitoring
+#### Installation example 
+
+You can install CRDs and default snapshot controller by running following commands:
```

Comment:
Change "following" to "these"

Reply:
Done.

Reply:
@adarsh-dell : could u address the comment here